Gordon360/Controllers/ProfilesController.cs

```diff
@@ -466,6 +485,36 @@ public async Task<ActionResult<FacultyStaffProfileViewModel>> UpdateOfficeHours(
         return Ok(result);
     }
 
+    /// <summary>
+    /// Set visibility of some piece of personal data for user.
+    /// </summary>
+    /// <param name="userPrivacy">Faculty Staff Privacy Decisions (see UserPrivacyUpdateViewModel)</param>
+    /// <returns></returns>
+    [HttpPut]
+    [Route("user_privacy")]
+    // [StateYourBusiness(operation = Operation.UPDATE, resource = Resource.PROFILE_PRIVACY)]
+    public async Task<ActionResult<UserPrivacyUpdateViewModel>> UpdateUserPrivacyAsync(UserPrivacyUpdateViewModel userPrivacy)
+    {
+        var authenticatedUserUsername = AuthUtils.GetUsername(User);
+        await profileService.UpdateUserPrivacyAsync(authenticatedUserUsername, userPrivacy);
+        return Ok();
+    }
+
+    /// <summary>
+    /// Return a list visibility groups.
+    /// </summary>
+    /// <returns> All visibility groups (Public, FacStaff, Private)</returns>
+    [HttpGet]
+    [Route("visibility_group")]
+    public ActionResult<IEnumerable<string>> GetVisibilityGroup()
+    {
+        var groups = context.UserPrivacy_Visibility_Groups.Select(up_v_g => up_v_g.Group)
+                               .Distinct()
+                               .Where(g => g != null)
+                               .OrderBy(g => g);
```

Comment:
```suggestion
                               .OrderBy(g => g);
```
I'm pretty sure this does nothing